exporter/SynthesisFusionAddin/src/Parser/ExporterOptions.py

```diff
@@ -77,7 +77,13 @@ class ModelHierarchy(Enum):
 
 @dataclass
 class ExporterOptions:
-    fileLocation: str = field(default=(os.getenv("HOME") if platform.system() == "Windows" else os.path.expanduser("~")))
+    fileLocation: str = field(
+        default=(
+            os.getenv("HOME")
+            if platform.system() == "Windows"
+            else os.path.expanduser("~")
+        )
+    )
```

Comment:
> [!warning]
> - This change will have conflicts with #985.

Reply:
Can we wait for that to be merged, then rerun the formatter on this branch?